tests/contrib/test_ensemble.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from autogoal.contrib.ensemble._stacking import StackingEnsemble
+
+def test_stacking(): 
+    X_train = [[0, 0], [1, 1]]
+    y_train = [0, 1]
+    X_test = [[2.,2.]]
+    ensemble = StackingEnsemble([SVC(),GaussianNB()], [SVC()])
+    print(ensemble.forward_pass(X_train, y_train))
```

Comment:
Now here what you want to do to have an actual test case is to include an assertion, something like:

```
assert y_pred == y_test
```

Or maybe `np.abs(y_pred - y_test).mean() < 0.01` or something like that.